examples/pxScene2d/src/pxServiceManager.cpp

```diff
@@ -0,0 +1,29 @@
+#ifdef USE_RT_REMOTE
```

Comment:
We no longer need to surround the whole source file with a #ifdef since pxServiceManager will only be compiled in conditionally based on values passed to the Makefile.  Please refer to the comments above.